homeassistant/components/freebox/switch.py

```diff
@@ -0,0 +1,62 @@
+"""Support for Freebox Delta, Revolution and Mini 4K."""
+import logging
+
+from homeassistant.components.freebox import DATA_FREEBOX
```

Comment:
Sorry, there is a new requirement that imports of own files should now be relative, home-assistant/architecture#180.
```suggestion
from . import DATA_FREEBOX
```

Reply:
changed it ! thx